.github/workflows/licensing.yml

```diff
@@ -40,3 +40,54 @@ jobs:
         git config --global --add safe.directory "$GITHUB_WORKSPACE"
         reuse --version
         reuse spdx | ./scripts/spdx_checks.py
+
+    - name: Checkout target
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4
+      with:
+        ref: ${{ github.event.pull_request.base.sha }}
+
+# REUSE-IgnoreStart
+    - name: Copyright dates
+      env:
+        BASE_REF: ${{ github.event.pull_request.base.sha }}
```

Comment:
Is this going to have the same problem as `templates-dyff` in that this commit is the current HEAD of the target branch rather than most-recent common ancestor of the PR branch and the target branch?

Reply:
No. Because `git diff A...B` is the same as `git diff $(git merge-base A B) B` and `git merge-base A B` returns the most recent common ancestor for A & B.

#450 uses `git merge-base` to fix the `template-dyff` problem